### DISCUSSION TITLE
Problem with button actions since the new version

### DISCUSSION THREAD

**Discussion by JGalbois** [GrapesJS/grapesjs#4398] [General]
Since the latest version 0.19.4 my trait button doesn't fire.

```
 type: 'button', // Type of the trait
            text: 'Properties ...',
				  id : 'TRproperties',
				  name: 'trparams',
				  full: true, // Full width button
				  command: function (edit,btn){
						console.log("Btn property", tAttrib.trid)
						
					
				  },
```
A see the button, but no event when i click on.

Thank's for your help

**Comment by artf** [comment]
Yeah, it was already reported [here](https://github.com/artf/grapesjs/issues/4388) and the fix is ready for the next release.
